libs/src/api/entityManager.ts

```diff
@@ -180,7 +180,7 @@ export class EntityManager extends Base {
         action: updateAction, // why include action here?
         entityType,
         playlist_id: playlistId,
-        playlist_contents: trackIds || playlist.playlist_contents,
+        playlist_contents: trackIds || playlist.playlist_contents.trackIds, // TODO fix schema validation from array to object
```

Comment:
I handled this in base branch - we're going to keep it as an array and do the diffing logic indexing side. makes things cleaner re: notifs etc. let's chat offline